charts/asset-manager/templates/worker-deployment.yaml

```diff
@@ -69,6 +69,9 @@ spec:
           volumeMounts:
             - name: clam-virus-db
               mountPath: /var/lib/clamav
+            - name: {{ .Release.Name }}-clamd-conf
+              mountPath: /etc/clamav/clamd.conf
+              subPath: clamd.conf
```

Comment:
It's best not to use subPath because this prevents updating the configmap.

Reply:
I dunno to what extent we care in this case though. Not saying you need to change it, just something to be aware of when using it. If the dir is empty anyway apart from this file then I would get rid of the subPath and just mount as `/etc/clamav`, but if not then maybe it's not worth the faff.

Reply:
ok, let me see what's in the volume.

Reply:
other stuff inside unfortunately.